build_compilers/lib/src/summary_builder.dart

```diff
@@ -76,7 +76,7 @@ Future createUnlinkedSummary(Module module, BuildStep buildStep,
   request.arguments.addAll([
     '--build-summary-only',
     '--build-summary-only-unlinked',
-    '--build-summary-output=${summaryOutputFile.path}',
+    '--build-summary-output-semantic=${summaryOutputFile.path}',
```

Comment:
At least a few of these changes feel like they could be justified in isolation and should be relatively easy to move to a new PR.

Can we submit this one separately?

Reply:
Do we get a speedup w/ this?

Reply:
I will break this out into its own cl and test in isolation, its mostly about invalidating later build steps but it should also improve the time to read and write summaries